docs/decisions/0002-text-editor-adr.md

```diff
@@ -0,0 +1,86 @@
+# Status
+
+Accepted
+
+# Context
+
+We seek to hoist the legacy text (HTML) text editor out of the monolith into a decoupled React application
+without reducing editor capabilities in the process.
+
+The legacy editor delegates the bulk of its operations to the tinyMCE editor
+
+
+
+# Decision set (each decision small)
+
+1. Rely on editor-wide Blockstore networking utilities for loading from, and saving to, XBlock content
+2. Continue using the tinyMCE editor
+3. Wrap the tinyMCE editor so as to be able to handle it as a vanilla React component
+4. Use a mix of configuration and tinyMCE API calls to customize the editor as needed 
+    * (e.g., offer image upload and image gallery selection)
+
+
+
+# Consequences
+
+## Rely on editor-wide BlockStore networking decision
```

Comment:
This section, although it adroitly captures the request layer and the general network abstraction, might be best served as its own adr, as none of the information you captured is unique to the text editor.